packages/appcd-default-plugins/src/index.js

```diff
@@ -0,0 +1,242 @@
+import appcdLogger from 'appcd-logger';
+import fs from 'fs-extra';
+import globule from 'globule';
+import os from 'os';
+import pacote from 'pacote';
+import path from 'path';
+import semver from 'semver';
+
+import { spawn } from 'child_process';
+
+const logger = appcdLogger('appcd:default-plugins');
+const { highlight } = appcdLogger.styles;
+
+/**
+ * ?
+ *
+ * @param {String} pluginsDir - Path to the plugins directory.
+ * @returns {Promise}
+ */
+export default async function installDefaultPlugins(pluginsDir) {
+	const start = new Date();
+
+	if (!pluginsDir || typeof pluginsDir !== 'string') {
+		throw new TypeError('Expected plugins directory to be a non-empty string');
+	}
+
+	// find yarn and lerna
+	const yarn = find('yarn');
+	if (yarn) {
+		logger.log(`Found yarn: ${highlight(yarn)}`);
+	} else {
+		logger.error('Unable to find yarn bin, skipping install default plugins');
+		return;
+	}
+
+	const lerna = find('lerna');
+	if (lerna) {
+		logger.log(`Found lerna: ${highlight(lerna)}`);
+	} else {
+		logger.error('Unable to find lerna bin, skipping install default plugins');
+		return;
+	}
+
+	const packagesDir = path.join(pluginsDir, 'packages');
+	const invalidDir = path.join(pluginsDir, 'invalid');
+	const linksDir = path.join(os.homedir(), '.config', 'yarn', 'link');
+	const workspaces = [];
+	const { plugins } = fs.readJsonSync(path.resolve(__dirname, '..', 'package.json'));
```

Comment:
Maybe use the promise version other *Sync for this and other fs methods throughtout

Reply:
Fixed!